test/ejabberd_tests/tests/rest_client_SUITE.erl

```diff
@@ -61,171 +61,205 @@ end_per_testcase(TC, C) ->
     escalus:end_per_testcase(TC, C).
 
 msg_is_sent_and_delivered(Config) ->
-    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
-        M = send_message(alice, Alice, Bob),
-        Msg = escalus:wait_for_stanza(Bob),
-        escalus:assert(is_chat_message, [maps:get(body, M)], Msg)
-    end).
+    F = fun(Alice, Bob) ->
```

Comment:
The previous way (fun as an arg story or fresh_story) was correct. I know that it confuses some auto-indenters, but we decided in this particular case (providing fun to story or fresh_story) can be indented as it was here.